tools/scripts/release/auto_build.py

```diff
@@ -299,20 +300,13 @@ def build_all(self):
                 config["path"] = os.path.join(tmpdirname, get_name_hash(name, 16))
                 os.mkdir(config["path"])
 
-                # Cache sign config value
-                old_sign = self.sign
-
-                # Don't use signature support for non-prod bootloaders so they'll
-                # accept any image
-                if self.sign and (name == "bootloader") and not self.is_prod:
-                    print("Disabling signature support for bootloader")
-                    self.sign = False
+                if "sign" not in config:
+                    config["sign"] = False
+                if "encrypt" not in config:
+                    config["encrypt"] = False
```

Comment:
The `sign` and `encrypt` keys are optional in the yml config. In our current version of default.yml, `sign` is always explicitly set to true or false, and we're not using `encrypt` at all.